CHANGES_NEXT_RELEASE

```diff
@@ -22,6 +22,8 @@
 - Fix: A bug fixed in URI param 'q', avoiding false matches when using string match operator (~=) that matches AND some operator that does not match.
```

Comment:
Just for not forgetting: have a look to FIXME "1068" on MongoCommonUpdate.cpp file.